drivers/led/Kconfig.pca_common

```diff
@@ -17,3 +17,13 @@ config PCA_COMMON
 	   * PCA9633 with 4 channels.
 	   * PCA9634 with 8 channels.
 	   * TLC59108 with 8 channels.
+
+if PCA_COMMON
+
+config PCA_COMMON_EXTERNAL_DRIVER
+	bool "Set external driver on startup"
+	depends on $(dt_compat_enabled,$(DT_COMPAT_NXP_PCA9634))
+	help
+	  Set external driver option, configured in dts, on startup.
```

Comment:
If the external-driver property is actually defined in DT, is there a reason to allow to disable external driver support from Kconfig ? If no, then maybe this option is not needed at all.

Reply:
I have removed Kconfig settings and replaced them it by `DT_PROP_ANY` macro.